sklearn/svm/_classes.py

```diff
@@ -1334,6 +1334,10 @@ class OneClassSVM(OutlierMixin, BaseLibSVM):
     array([-1,  1,  1,  1, -1])
     >>> clf.score_samples(X)
     array([1.7798..., 2.0547..., 2.0556..., 2.0561..., 1.7332...])
+
+    See also
+    --------
+    sklearn.linear_model.SGDOneClassSVM
```

Comment:
Maybe add a quick note to point out the favorable linear complexity for datasets with large number of samples.

Here is an example of such notes in a see also section: 

https://github.com/scikit-learn/scikit-learn/blob/95119c13a/sklearn/ensemble/_forest.py#L1727-L1731